### ISSUE TITLE
Can not start on raspberrypi

### ISSUE THREAD

**Issue by felixletsplayYT** [OpenRCT2/OpenRCT2#5696]
**OS:Linux(raspbian minimal)** 
**Version: 0.0.8** 
**Commit/Build:050ac3d** 

It will not start: pi@raspberrypi:~/OpenRCT2 $ ./openrct2 host world.sv6 --headless
bash: ./openrct2: cannot execute binary file

**Steps to reproduce:**
1.  Try to start

file shows: 
openrct2: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 2.6.32, BuildID[sha1]=608953bec001af5ba7c6a7dcc5f0cec46687e35e, not stripped

So my Question, is there any posibility to port this for x86.
Thank You

**Comment by janisozaur** [comment]
Yes ARMv7 builds were tested numerous times and work just as well as x86 and x86-64 ones, but we're not going to provide the builds for it ourselves. You have to roll your own.

**Comment by felixletsplayYT** [comment]
How do I build ARM

**Comment by janisozaur** [comment]
Same way as on x86

**Comment by rwjuk** [comment]
I'd recommend cross-compiling for ARM on your regular (assuming x86) PC, as compilation on a Raspberry Pi will be very slow.
